posts/post-1.md

```diff
@@ -1,10 +1,34 @@
+
+Dan is a local dairy farmer and his small farm produces the best milk in in town. He can't easily sell the production at a reasonable price directly to end consumers but only sell to wholesalers in a much cheaper price. Dan is not satisfied with his situation and wants to promote his products directly to local consumers.
+
+From now on, people like Cindy and Dan will not be bothered by their inconveniences any more thanks to **Virto Network**. Virto Network implements a clever solution to giving **utility to cryptocurrencies** and fair **trade without intermediaries**  - the *Local Incentives Protocol* which enables blockchain based **De-Commerce** infrastructure:
+
+- For merchants, they can either easily create their own **marketplaces**, e.g. a grocery shop, to directly serve consumers avoiding the extra cost of intermediaries or together form **marketplaces**, e.g. a Sunday Fleamarket to scale up their service to consumers. All needed is for marketplace builders to connect their applications to the familiar Web2-like APIs offered by Virto, as simply as using **Shopify**, but almost free.
+
+- For users, they can conveniently use their crypto assets to buy any products or services offered by the different marketplaces, including trading with fiat money. It means, users could not only, e.g. buy their favorite book, but also easily swap their digital assets for fiat money and back. We believe this makes Virto unique by providing a bridge between the future blockchain metaverses and real society.
+
+ >  **Swap.Cash**, one of the core team's reference applications will be launched early this year, first using an existing Kusama parachain like Karura before it's finally deployed in Virto Network by Q2 2022.
+
+Amazon, Ebay, Alibaba these global e-commerce platforms have been steadily increasing in the past decades, which offers opportunities to integrate millions of marketplaces. Meanwhile more and more critics have also arose, especially concentrating on their anti-competitive or monopolistic behaviors, which result in price control, price discrimination and employee management. These however would never be possible in a decentralized system backed up by blockchain technology, in contrary, all these missing valuable virtues, **fairness, transparency, efficiency** will be better respected and valued in **de-commerce** systems  as they are rooted in the its design.
+
+In addition, **Local Incentives Protocol** also incentives the formations of **Local Communityies** that function as **DAOs: Decentralised Autonomous Organisation** by reserving local tax, which is collected from the transactions and is only at the disposal of the people from the community. The community is governed by its own people. This will also help the system fight against resource monopoly within the system and support local economy.
```

Comment:
```suggestion
In addition, **Local Incentives Protocol** also incentives the formations of **Local Communities** that function as **DAOs: Decentralised Autonomous Organisation** by reserving local tax, which is collected from the transactions and is only at the disposal of the people from the community. The community is governed by its own people. This will also help the system fight against resource monopoly within the system and support the local economy.
```